openjdk_regression/ProblemList_openjdk8-openj9.txt

```diff
@@ -0,0 +1,128 @@
+# List of tests that should not be run by adoptOpenjdk playlist:
+#   1. Does not run with jtreg -samevm mode
+#   2. Causes problems in jtreg -samevm mode for jtreg or tests that follow it
+#   3. The test is too slow or consumes too many system resources
+#   4. The test fails when run on any official build systems
+
+# More than one label is allowed but must be on the same line.
+#
+#############################################################################
+#
+# Fixing the tests:
+#
+# Some tests just may need to be run with "othervm", and that can easily be
+#   done my adding a @run line (or modifying any existing @run):
+#      @run main/othervm NameOfMainClass
+#   Make sure this @run follows any use of @library.
+#   Otherwise, if the test is a samevm possibility, make sure the test is
+#     cleaning up after itself, closing all streams, deleting temp files, etc.
+#
+# Keep in mind that the bug could be in many places, and even different per
+#   platform, it could be a bug in any one of:
+#      - the testcase
+#      - the jdk (jdk classes, native code, or hotspot)
+#      - the native compiler
+#      - the javac compiler
+#      - the OS (depends on what the testcase does)
+#
+# If you managed to really fix one of these tests, here is how you can
+#    remove tests from this list:
+#  1. Make sure test passes on all platforms with samevm, or mark it othervm
+#  2. Make sure test passes on all platforms when run with it's entire group
+#  3. Make sure both VMs are tested, -server and -client, if possible
+#  4. Make sure you try the -d64 option on Solaris
+#  5. Use a tool like JPRT or something to verify these results
```

Comment:
Quite sure that some of these instructions are not currently possible or correct to do at AdoptOpenJDK, but I realize they are a carry-over of the set of instructions from OpenJDK.  Some instructions are vague and offer no real guidance as to what actual actions should happen next.  At some point we should revisit the instructions in this file, but that is a whole other issue for another day.  I will not block this PR for it.

Reply:
Yes, need to revisit it later